iphone/Maps/Core/iCloud/DefaultLocalDirectoryMonitor.swift

```diff
@@ -63,10 +63,11 @@ final class DefaultLocalDirectoryMonitor: LocalDirectoryMonitor {
     guard state != .started else { return }
 
     let nowTimer = Timer.scheduledTimer(withTimeInterval: .zero, repeats: false) { [weak self] _ in
-      LOG(.debug, "LocalMonitor: Initial timer firing...")
+      LOG(.debug, "Initial timer firing...")
```

Comment:
Is there any mention of the iCloud in this log line? Should logs be tagged somehow? E.g. "iCloud: Initial timer firing"?

Reply:
I removed all the mentions because the c++ log composer add the class name and the current method to the log message:
```
I(1) 509.95730 SynchronizationStateManager.swift:78 resolveEvent(_:): Cloud content: 
File path: /private/var/mobile/Library/Mobile Documents/iCloud~app~organicmaps/Documents/doc1.kml, lastModified: 1722530601.0
File path: /private/var/mobile/Library/Mobile Documents/iCloud~app~organicmaps/Documents/filesFrom Mukhuri to Tobavarchkhili lake and further to Svaneti1.kml, lastModified: 1722530601.0
File path: /private/var/mobile/Library/Mobile Documents/iCloud~app~organicmaps/Documents/filesMy Places.kml, lastModified: 1722530601.0
```

There is no need for this additional mentions.